js/api.md

```diff
@@ -1,23 +1,48 @@
 ---
+title: API
 ---
+{% if jekyll.environment == 'production' %}
+  {% assign base_dir = site.amplify.docs_baseurl %}
+{% endif %}
+{% assign images_base = base_dir | append: page.dir | append: re"images" %}
+
 # API
 
-The API category provides a solution for making HTTP requests to REST and GraphQL endpoints. It includes a [AWS Signature Version 4](http://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signer class which automatically signs all AWS API requests for you.
+The API category provides a solution for making HTTP requests to REST and GraphQL endpoints. It includes a [AWS Signature Version 4](http://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signer class which automatically signs all AWS API requests for you as well as methods to use API Keys, Amazon Cognito User Pools, or 3rd party OIDC providers.
+
 
 ## Using GraphQL Endpoints
 
-AWS Amplify API Module supports AWS AppSync or any other GraphQL backend.
+AWS Amplify API Module supports AWS AppSync or any other GraphQL backends.
 
 To learn more about GraphQL, please visit [GraphQL website](http://graphql.org/learn/).
 {: .callout .callout--action}
 
 ### Using AWS AppSync
 
-AWS AppSync is a cloud-based fully-managed GraphQL service that is integrated with AWS Amplify. You can create your AWS AppSync GraphQL API backend with the Amplify CLI and connect to your backend using AWS Amplify API category. 
-
-Learn more about [AWS AppSync](https://aws.amazon.com/appsync/) by visiting [AWS AppSync Developer Guide](https://docs.aws.amazon.com/appsync/latest/devguide/welcome.html){: .target='new'}.
+AWS AppSync helps you build data-driven apps with real-time and offline capabilities. Learn more about [AWS AppSync](https://aws.amazon.com/appsync/) by visiting [AWS AppSync Developer Guide](https://docs.aws.amazon.com/appsync/latest/devguide/welcome.html){: .target='new'}.
 {: .callout .callout--action}
 
+
+Amplify Framework offers two client options for AppSync:
+
+- [Amplify GraphQL client](./api#amplify-graphql-client)
+
+- [AWS AppSync SDK](https://github.com/awslabs/aws-mobile-appsync-sdk-js/)
+
+
+The [AppSync Apollo client SDK](https://github.com/awslabs/aws-mobile-appsync-sdk-js/) enables you to integrate your app with the AWS AppSync service and integrates with the Apollo client found [here](https://github.com/apollographql/apollo-client/). The SDK supports multiple authorization models, handles subscription handshake protocols for real-time updates to data, and has built-in capabilities for offline support that makes it easy to integrate into your app.
```

Comment:
Should this also be changed from "AppSync Apollo client SDK" to "AWS AppSync SDK"?